entityTypes.js

```diff
@@ -19,6 +31,10 @@ export default [
     "label": "Limited Liability Company",
     "value": "LLC"
   },
+  {
+    "label": " Professional Limited Liability Company",
```

Comment:
remove whitespace

Reply:
Removed in this [commit](https://github.com/digitalbazaar/bedrock-vue-organization-form/compare/6908b84aa592a97915280e1cc40ea4a5698a58f2..e34be51ecf7a08c1893a7ff29b94f0d38ef659f8).